pal/src/tlsio_arduino.c

```diff
@@ -408,10 +426,11 @@ static void dowork_read(TLS_IO_INSTANCE* tls_io_instance)
                     enter_tlsio_error_state(tls_io_instance);
                 }
                 else
-                {
-                    /* Codes_SRS_TLSIO_30_102: [ If the TLS connection receives no data then tlsio_dowork shall not call the on_bytes_received
-                    callback. ]*/
-                }
+		{
```

Comment:
still have tabs in this file. please use the VSCode regex search for `\t` and change to four spaces.

Reply:
Resolved.